h/api/models.py

```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+
+from ..models import Annotation  # pylint: disable-msg=import-self
```

Comment:
You can remove these disables. I don't get these on newer versions of pylint, at least with my more-lint branch I don't.